source/install-desktops/customize-desktop-app.rst

```diff
@@ -0,0 +1,34 @@
+.. _install-desktops-customize-desktop-app:
+
+Customize Desktop App
+=====================
+
+In some cases the defaults used to submit a batch job may not work on your
+given cluster or you may just want to remove some form options presented to the
+user and fill in hard-coded values. All of these cases and more can easily be
+customized in the following files:
+
+:file:`bc_desktop/local/{cluster_desktop}.yml`
+  describes a given desktop app along with form attributes presented to the
```

Comment:
Hah, guess I lowercased it because it is "a" desktop app and not "the" Desktop App. But I am fine upper casing it.